tsconfig-base.json

```diff
@@ -13,7 +13,7 @@
 
     "target": "es2020",
     "module": "ESNext",
-    "lib": ["ES2024.Object"],
+    "lib": ["ES2024.Object", "ES2023.Array"],
```

Comment:
I assume this is for findLast. Seems good enough usage for me, https://caniuse.com/mdn-javascript_builtins_array_findlast.